src/contributors.ts

```diff
@@ -137,8 +137,8 @@ export class Contributors {
                     user.user.id
                 );
 
-                if ('avatarUrl' in crowdinMember.data && crowdinMember.data.avatarUrl) {
```

Comment:
I'd suggest keeping this condition and trying to set `picture = user.user.avatarUrl` before the try-catch block if the `'avatarUrl' in user.user` exists

Reply:
ohh, i'll update it now

Reply:
done

Reply:
@RealRichi3 the dist check WF still failing https://github.com/andrii-bodnar/action-crowdin-contributors/actions/runs/5313195576/jobs/9619077844?pr=56

could you please take a look?

Reply:
Make sure you're running the same Node version locally as in the CI workflow - Node 16.x